app/presenters/task_list_presenter.rb

```diff
@@ -1,5 +1,5 @@
 class TaskListPresenter
-  TaskListRow = Struct.new(:title, :minutes_to_complete, :value, :start_path)
+  TaskListRow = Struct.new(:title, :minutes_to_complete, :value, :start_path, :can_start?)
```

Comment:
maybe change 'can_start?' to 'show_start_button?' - it's more accurate/meaningful